FogCreek/Glitch-Community
# Lime hawthorn: hookify collection editor

## Links
* Remix link: https://lime-hawthorn.glitch.me

## Changes:
* replace CollectionEditor component with useCollectionEditor hook
* clean up Collection and Category pages

## How To Test:
* log into https://lime-hawthorn.glitch.me
* verify that you can reorder items in one of your collections
* verify that you can add a project from https://lime-hawthorn.glitch.me/hello-worlds to a collection
